templates/rocketchat-migrate.nomad

```diff
@@ -14,26 +14,26 @@ job "rocketchat-migrate" {
       ${ task_logs() }
 
       driver = "docker"
-      config = {
+      config {
         image = "${config.image('liquidinvestigations/caboose')}"
         args = ["python", "/local/rocketchat-caboose.py"]
         labels {
           liquid_task = "rocketchat-caboose"
         }
       }
       template {
-        data = <<EOF
+        data = <<-EOF
 # Auto-generated by rocketchat migrate script
```

Comment:
We can indent this chunk now, correct?

Reply:
Did some more testing and unfortunately no. I suppose the env parser is doing something behind the scenes when the dash is set, because it doesn't work for that python script for example. Also to make the changes with `-EOF` clearer: The generated file takes the indentation of `data = <<-EOF` as reference, so any tab and space apart from there is still in the file. Following example:
```
       template {
        data = <<-EOF
          {{- range service "rocketchat-mongo" }}
            MONGO_ADDRESS = {{.Address | toJSON }}
            MONGO_PORT = {{.Port | toJSON }}
        {{- end }}
        EOF
        destination = "local/liquid.env"
        env = true
      }
```
results in:
```

    MONGO_ADDRESS = "10.66.60.1"
    MONGO_PORT = 20615
```
we have 2 space indentation to the service check + 2 inside the service check.
with `<<EOF`, this will be generated:
```

            MONGO_ADDRESS = "10.66.60.1"
            MONGO_PORT = 30816
        
```
The last line has 8 spaces, just as the line with `EOF` on.
```
      template {
        data = <<-EOF
        {{- range service "rocketchat-mongo" }}
        MONGO_ADDRESS = {{.Address | toJSON }}
        MONGO_PORT = {{.Port | toJSON }}
        {{- end }}
        EOF
        destination = "local/liquid.env"
        env = true
      }
```
will generate an env file with no spaces, omitting `-` will result in an env file with (you guessed it) 8 spaces.

Reply:
Ok, we can live with same-level indentation. So let's indent this block to be even with `data = <<-EOF`.